### DISCUSSION TITLE
Twitter Link not working as a URL

### DISCUSSION THREAD

**Discussion by Humiddifier** [mozilla/hubs#3625] [Spoke Questions] [answered]
Hi all, I am trying to attach a link to artists on Twitter so that they can be properly credited. The Instagram links are working how intended, click the link element, and you get sent to their post of the art. But the Twitter links keep popping up as a broken link, even though they are linking to a specific page on a user's Twitter page. I could really use some support with this so we can properly credit them in the art show. 

Thank you.
Humiddifier

Here are some pictures of the problem I am having.
<img width="1415" alt="Screen Shot 2020-12-29 at 1 20 35 PM" src="https://user-images.githubusercontent.com/76749527/103308718-b8601400-49d8-11eb-821c-9b9ce4b53001.png">
<img width="1403" alt="Screen Shot 2020-12-29 at 1 19 02 PM" src="https://user-images.githubusercontent.com/76749527/103308727-bb5b0480-49d8-11eb-80ab-d4075681e397.png">

**Comment by robinkwilson** [comment, marked as answer]
Ah yes! This is a known bug: the parent issue is https://github.com/mozilla/reticulum/issues/435 and the twitter specific issue is https://github.com/mozilla/hubs/issues/3654

Follow this for updates, I unfortunately can't guarantee an exact fix date but I know we're talking about these issues next week.
